csm_web/frontend/src/components/resource_aggregation/ResourceEdit.tsx

```diff
@@ -183,7 +279,7 @@ export const ResourceEdit = ({ resource, onChange, onSubmit, onCancel }: Resourc
 
     // validate all fields
     if (validate(true)) {
-      onSubmit(e, existingWorksheetMap, newWorksheets);
+      onSubmit(e, existingWorksheetMap, existingLinkMap, newWorksheets, newLinks);
```

Comment:
Currently, submitting a link without https gets past frontend validation but results in a bad request when sent to the backend. Make sure to send the full link to the backend rather than what is just input by the user.